pkg/database/truncator_test.go

```diff
@@ -92,7 +92,7 @@ func Test_vlogCompactor_WithMultipleIO(t *testing.T) {
 	hdr, err := db.st.ReadTxHeader(deletePointTx, false)
 	require.NoError(t, err)
 	c := NewVlogTruncator(db)
-	require.NoError(t, c.Truncate(hdr.ID))
+	require.NoError(t, c.Truncate(context.TODO(), hdr.ID))
```

Comment:
`context.Background()` may be used instead of `context.TODO()`, here and in other places